create_magic.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+ARCH=x86_64
+TYPE=qcow2
+DIR=~/build
+mkdir $DIR
+DIST=fedora21
+DOMAIN=$DIST-cockpit
```

Comment:
What is this "DIST-cockpit"? That looks like we're hard-coding a libvirt domain name?